projects/packages/sync/src/class-defaults.php

```diff
@@ -1329,6 +1329,10 @@ public static function is_multi_network() {
 			'chunk_size' => 100,
 			'max_chunks' => 10,
 		),
+		'woocommerce'        => array(
```

Comment:
I'm a bit concerned around having a discrepancy between the defaults vs the current DB setting.
This can produce a Warning in `send_full_sync_actions` upon fetching the limits: `$limits = Settings::get_setting( 'full_sync_limits' )[ $this->name() ];`
I think we should consider checking if the key (module name) is present in the stored settings, then fallback to the defaults so that we don't get: `PHP Warning:  Undefined array key "woocommerce" in /usr/local/src/jetpack-monorepo/projects/packages/sync/src/modules/class-module.php on line 384` and `PHP Warning:  Trying to access array offset on value of type null in /usr/local/src/jetpack-monorepo/projects/packages/sync/src/modules/class-module.php on line 390`

Reply:
That's a fair point. I wrongly assumed that the defaults were going to update the settings. 
I did add a change in [e32ac11](https://github.com/Automattic/jetpack/pull/39254/commits/e32ac116f661c88d9e9dfcb281927c2824d662aa) to either pick from default settings or use a default array